bin/cli.js

```diff
@@ -86,6 +86,9 @@ process.on("exit", () => {
             } else {
                 configObj = {};
             }
+        } else if (configObj === undefined) {
+            logger.error("Failed to get default export from configuration file");
```

Comment:
This branch is only reached when there is no default export, correct? Maybe we can make the message more explicit about that ("failed to get ..." reads to me like it's a problem with the program, not the config). How about:

```suggestion
            logger.error("Default export missing from configuration file (use `export default {...}` or `module.exports = {...}`");
```

Reply:
Great, applied your suggestion 👍🏼